### PR TITLE
feat: added partner showcase component

### PR DESCRIPTION
**Pull request by mayur-satav**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:

- Before: https://main--pricefx-eds--pricefx.hlx.live/
- After: https://feature-pai-30-be-partner-showcase--pricefx-eds--pricefx.hlx.live/

### CODE REVIEW

File: helix-query.yaml [hunk 1]
```diff
@@ -41,3 +41,30 @@ indices:
       cq-tags:
         select: head > meta[name="cq-tags"]
         value: attribute(el, "content")
+  partners:
+    include:
+      - /partner/**
```
Comment (amogh-daryapurkar) [inline helix-query.yaml:46]: @mayur-satav Can you confirm if this is partner or partners?

**Review by amogh-daryapurkar — COMMENTED** [review]
@mayur-satav Can you check if the url is partner or partners?

File: helix-query.yaml (hunk 1 above)
Comment (mayur-satav) [reply helix-query.yaml:46]: i think as per the screenshot Munnesh shared it is the partners

Comment (amogh-daryapurkar) [reply helix-query.yaml:46]: @muneeskumar-rajendran Can you confirm the same? 

@mayur-satav Can you update it to partners if @muneeskumar-rajendran confirms that?

**Review by muneeskumar-rajendran — APPROVED** [review]

File: component-models.json
Comment (amogh-daryapurkar) [inline component-models.json:1]: @mayur-satav We don't need Filter by Platforms option which is present in design. Can you confirm if you considered it here ? If not, we are good !

**Review by muneeskumar-rajendran — APPROVED** [review]

**Pull request merged**